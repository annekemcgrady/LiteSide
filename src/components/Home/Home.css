.home {
  background-color: #2f2f2f;
  border: 1px solid #ffff2b54;
  box-sizing: border-box;
  color: #ffff2b;
  height: 90vh;
  width: 50%;
  font-size: 1.5em;
  padding-top: 20%;
  }

  