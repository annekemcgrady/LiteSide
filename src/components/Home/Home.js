import React from 'react';
import './Home.css'


const Home = () => {

  return (

  <aside className="home">
    <p>Welcome to LITE SIDE, please choose a category</p>
  </aside>
  )

      
}

export default Home;