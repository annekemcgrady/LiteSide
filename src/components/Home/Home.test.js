import React from 'react';
import { shallow, mount } from 'enzyme';
import Home from './Home.js';

describe('Home', ()=> {

  it('should match the snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  })

})