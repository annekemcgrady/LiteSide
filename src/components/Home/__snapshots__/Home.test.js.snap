// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match the snapshot 1`] = `
<aside
  className="home"
>
  <p>
    Welcome to LITE SIDE, please choose a category
  </p>
</aside>
`;
